testutils/utils.go

```diff
@@ -244,9 +278,9 @@ func RunCNIPluginWithId(netconf, k8sName, ip, containerId string, targetNs ns.Ne
 		panic("some error found")
 	}
 
-	io.WriteString(stdin, netconf)
-	io.WriteString(stdin, "\n")
-	stdin.Close()
+	_, _ = io.WriteString(stdin, netconf)
```

Comment:
not sure why you changed this.

Reply:
unless you plan to do some Expect or err checking then best left as it.

Reply:
this is not refactoring. We have static error checking as part of the CI so it fails unless all the errors are handled or ignored, and it only checks the files that changed, and it was added to the go-build container recently that's why no one hit it until now

Reply:
ok, so if we're being forced to change due to CI - should we not be asserting something about the return values?